Storage/Storage/Model/Order+CoreDataProperties.swift

```diff
@@ -45,15 +45,14 @@ extension Order {
     @NSManaged public var shippingState: String?
     @NSManaged public var shippingTax: String?
     @NSManaged public var shippingTotal: String?
-    @NSManaged public var status: String
     @NSManaged public var total: String?
     @NSManaged public var totalTax: String?
     @NSManaged public var exclusiveForSearch: Bool
     @NSManaged public var coupons: Set<OrderCoupon>?
     @NSManaged public var items: Set<OrderItem>?
     @NSManaged public var notes: Set<OrderNote>?
     @NSManaged public var searchResults: Set<OrderSearchResults>?
-
+    @NSManaged public var status: String?
 }
 
 // MARK: Generated accessors for coupons
```

Comment:
Do I add accessors for OrderStatus? What's it for, and how do I write it, since OrderStatus isn't a to-many relationship?